### PR TITLE
e2e test case update for doctor

### PR DESCRIPTION
**Pull request by dakshaladia**

### CODE REVIEW

**Review by A-N1khil — APPROVED** [review]
LGTM!

File: client/cypress/e2e/DoctorPage.cy.js [hunk 1]
```diff
@@ -1,20 +1,52 @@
 describe("Doctor Page - End-to-End Tests", () => {
+    const doctorEmail = "amazing@slotter.com"; // Define the email to test
+    const doctorPassword = "Amazing@1"; // Replace with the actual password
```
Comment (A-N1khil) [inline client/cypress/e2e/DoctorPage.cy.js:3]: Can be picked from env to silence GitGuadian flag

Comment (A-N1khil) [reply client/cypress/e2e/DoctorPage.cy.js:3]: Resolve for merge, if needed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Contributing
+
```
Comment (snigdhansu) [inline CONTRIBUTING.md:2]: Remove file, not needed

Comment (A-N1khil) [reply CONTRIBUTING.md:2]: image

**Pull request merged**